code/workspaces/infrastructure-api/src/dataaccess/projectsRepository.js

```diff
@@ -30,7 +30,7 @@ async function createOrUpdate(project) {
 }
 
 async function deleteByKey(projectKey) {
-  return Project().remove({ key: projectKey }).exec();
+  return Project().deleteOne({ key: projectKey }).exec();
```

Comment:
Out of curiosity, how come the use of deleteOne over remove?

Reply:
I changed it to remove a deprecation warning

Reply:
Ahhh I see. Thanks for letting me know!